content/docs/fundamentals/converting/resolving-errors@pt_BR.md

```diff
@@ -0,0 +1,350 @@
+---
+$title: Corrigir erros de validação
+$order: 2
+toc: true
+---
+
+[TOC]
+
+Nesta seção, analisaremos e resolveremos os erros de validação de AMP na nossa página AMP.  Esses erros podem aparecer em outra ordem no seu console.
+
+## Incluir charset
+
+Para começar, corrigiremos o seguinte erro:
+
+<pre class="error-text">
+A tag obrigatória "meta charset=utf-8" está ausente ou incorreta.
+</pre>
+
+Para que o texto seja exibido corretamente, é necessário especificar o charset das páginas AMP. As informações do meta charset também precisam ser o primeiro filho da tag `<head> `. Essa tag precisa ser a primeira para evitar a reinterpretação de conteúdo adicionado antes da tag meta charset.
+
+**Adicione** o código a seguir como a primeira linha da tag `<head>`:
+
+
+```html
+<meta charset="utf-8" />
+```
+
+**Salve** o arquivo e atualize a página. Verifique se o erro de charset ainda ocorre.
+
+## Incluir link canônico
+
+Agora, veja este erro:
+
+<pre class="error-text">
+A tag obrigatória "link rel=canonical" está ausente ou incorreta.
+</pre>
+
+Todo documento AMP precisa ter um link que faça referência à respectiva versão "canônica".  Veja mais sobre o que é uma página canônica e quais são as diferentes abordagens para vincular o conteúdo canônico na etapa deste tutorial sobre [como fazer com que a página seja detectável](/pt_br/docs/fundamentals/converting/discoverable.html).
+
+Neste tutorial, o artigo HTML original que estamos convertendo será a página canônica.
+
+**Adicione** o código a seguir à tag `<meta charset="utf-8" />`:
+
+```html
+<link rel="canonical" href="/article.html">
+```
+
+{% call callout('Observação', type='note') %}
+É possível criar uma página AMP canônica independente. Nesse caso, o link canônico ainda é necessário, mas precisa levar ao próprio artigo AMP:
+
+```html
+<link rel="canonical" href="article.amp.html">
+```
+{% endcall %}
+
+Agora, **atualize** a página. Embora ainda haja muitos erros para corrigir, o erro do link canônico não está mais presente.
+
+## Especificar o atributo AMP
+
+A AMP exige um atributo no elemento raiz `<html>` das páginas para declarar que elas são documentos AMP.
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).